spec/features/user_signs_up_spec.rb

```diff
@@ -18,11 +18,15 @@
     fill_in 'Email', with: 'john@example.com'
     fill_in 'Password', with: 'password'
     fill_in 'Password confirmation', with: 'password'
+    attach_file "Profile Photo",
+      "#{Rails.root}/spec/support/images/unisex-bathroom-logo-md.png"
 
     click_button 'Sign up'
 
     expect(page).to have_content('Welcome! You have signed up successfully.')
     expect(page).to have_content('Sign Out')
+    expect(User.first.profile_photo.file.filename).to
+    eq("unisex-bathroom-logo-md.png")
```

Comment:
Since capybara deals with the view from the user's perspective, I would look for an image tag in the page with that particular filename.

```
expect(page).to have_css("img[src*='photo-name.png']")
```